frontend/src/pages/Login.jsx

```diff
@@ -247,17 +243,18 @@ export const Login = () => {
     }
   };
 
-
   return (
     <Container>
       <LeftColumn>
-        <StyledLogo src={logo} alt="logo" />
```

Comment:
if only the SVG works on all devices, why did you choose the other logo here?

Reply:
I tried to be consistent with using the logo component. As it is in use in every other part of the app. 
The logo component works everywhere, **except** in the specific place where I made the note (pointscard, probably because of the conditional rendering and some mysterious browser issue).